### DISCUSSION TITLE
A few keys for writers?

### DISCUSSION THREAD

**Discussion by ambrosefield** [argenos/hotkeysplus-obsidian#34] [Q&A] [answered]
Thank you for your wonderful plugin, it's a great help. If adding additional functionality, would you consider a couple of keys that would really help long-form writing in obsidian. Things like:

- **delete sentence** - from cursor to last full stop; 
- **delete to last punctuation mark**  from cursor ;
- **delete from previous full stop to current cursor**  and capitalize next word (useful for adjusting the starts of sentences)

etc,

Thank you for your consideration and work you've put into this.

**Comment by argenos** [comment]
Could you open an issue for that as a feature request? It's unlikely I will implement those as I use vim for that, but I'll make sure to add the help wanted tag to the issue you create in case someone is willing to add a PR for that.

**Comment by FelipeRearden** [comment, marked as answer]
@ambrosefield 

Maybe the commands of these two plugins help you in your workflow 🙏

https://github.com/timhor/obsidian-sentence-navigator#sentence-navigator

https://github.com/timhor/obsidian-editor-shortcuts#code-editor-shortcuts

Have a great day!

**Comment by ambrosefield** [comment]
Thank you all for your help.

Sentence navigator is fantastic and solves my issue.

Really appreciate this.

**Reply by FelipeRearden** [reply]
We are here to try to help each other :)

That'sthe foundation of the Obsidian community 


I'm happy to know that it's working for you 💙
